airgun/views/computeresource.py

```diff
@@ -68,9 +68,7 @@ class http_proxy(View):
 
     @provider_content.register('Google')
     class GCEProviderForm(View):
-        google_project_id = TextInput(id='compute_resource_project')
-        client_email = TextInput(id='compute_resource_email')
-        certificate_path = TextInput(id='compute_resource_key_path')
+        json_key = TextInput(id='gce_json')
```

Comment:
Yes, @Gauravtalreja1 we need to add that option as well but for the testing of the feature with the automation I have added a small update to make the cases work, will add the upload option, as well as @shweta83 , suggested above.

Reply:
@adarshdubey-star yes, I get that, but the ask from @shweta83 is just to include a locator for that button, so I think it can be covered in the same PR as well :)

Reply:
Added :)